core/build.gradle

```diff
@@ -5,11 +5,9 @@ buildscript {
         maven { url = "$repoUrl" }
         maven { url = "$springRepoUrl" }
         mavenCentral()
-        jcenter()
     }
     dependencies {
         classpath "org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}"
-        classpath "com.smokejumperit.gradle.license:Gradle-License-Report:0.0.2"
         classpath "gradle.plugin.com.github.jk1:gradle-license-report:0.3.4"
```

Comment:
What about this one? Is this possibly related to the one removed above?

Reply:
it is related to the jcenter removal (and it is used nowhere btw)

Reply:
Let me clarify. I was referring to this line:
```
classpath "gradle.plugin.com.github.jk1:gradle-license-report:0.3.4"
```
Should it be also removed in addition to line 12?

Reply:
it is used in core